hack/generator/pkg/conversions/testdata/CopyKnownReferenceProperty.golden

```diff
@@ -0,0 +1,33 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package vCurrent
+
+import (
+	"github.com/Azure/azure-service-operator/hack/generated/_apis/Verification/vNext"
+	"github.com/Azure/azure-service-operator/hack/generated/pkg/genruntime"
+)
+
+type Person struct {
+	KnownReference genruntime.KnownResourceReference `json:"known-reference"`
```

Comment:
minor: This should probably be `knownReference` in the JSON (even though it doesn't matter at all)